### PR TITLE
Generate site

### PR DESCRIPTION
**Pull request by duijf**
This PR adds static site generation based on the S3 stored state.

Contents:

 - Generate static site output based on S3 state
 - Rename HTML files in the `design` directory, convert to Jinja2, parameterize
 - Remove example images
 - Move assets to own directories
 - Catch some `boto3` exceptions that I ran into when testing

### CODE REVIEW

File: pxl/state.py [hunk 1]
```diff
@@ -31,18 +32,37 @@ def to_json(self):
 
 @dataclass
 class Album:
+    created: datetime.datetime
     images: List[Image]
     name_display: str
     name_nav: str
 
+    @property
+    def created_human(self):
```
Comment (HugoPeters1024) [inline pxl/state.py:41]: why the specific `human`? I assume that this refers to the timestamp it was created? Also it is not self evident to me what threadsafe issues have to do with this.

File: pxl/generate.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import pxl.state as state
+
+
+def build(overview: state.Overview,
+          output_dir: Path,
+          template_dir: Path,
+          bucket_puburl: str):
+    """Build a static site based on the state."""
+
+    index_template = load_template(template_dir / 'index.html.j2')
```
Comment (HugoPeters1024) [inline pxl/generate.py:15]: wut? this 4 real? pretty neat

**Review by HugoPeters1024 — CHANGES_REQUESTED** [review]
Please clarify the created locale comments, besides that it looks great!

File: pxl/generate.py (hunk 2 above)
Comment (duijf) [reply pxl/generate.py:15]: Yes

File: pxl/state.py (hunk 1 above)
Comment (duijf) [reply pxl/state.py:41]: I gave it this name because it is the timestamp for human consumption

The threadsafety stuff is just me ranting

**Pull request merged**